ruby/hashes.rb

```diff
@@ -0,0 +1,92 @@
+=begin
+Allow an interior designer to enter the details of a given client: the client's name, age, 
+number of children, decor theme, and so on \.
+(you can choose your own as long as it's a good mix of string, integer, and boolean data).
+
+Your program should ...
+
+Prompt the designer/user for all of this information.
+Convert any user input to the appropriate data type.
+Print the hash back out to the screen when the designer has answered all of the questions.
+Give the user the opportunity to update a key (no need to loop, once is fine). After all, sometimes users make mistakes! If the designer says "none", skip it. But if the designer enters "decor_theme" (for example), your program should ask for a new value and update the :decor_theme key. (Hint: Strings have methods that will turn them into symbols, which would be quite handy here.) You can assume the user will correctly input a key that exists in your hash -- no need to handle user errors.
+Print the latest version of the hash, and exit the program.
+
+	Ask the designer for information -method that returns answer
+	Create a database with the answers -method that creates and populates the database
+	Make edition possible -method that access the database and modifies items
+	Check proper type
+	User interfase that allows her to Create, Edit or Exit the program.	
+
+=end
+
+#GLOBAL VARIABLES
+database={}
+
+def ask question

```

Comment:
In general, the more accepted style is using `()` for methods, this is purely a stylistic comment.